### PR TITLE
fix problem setting passwords with chars like '('

### PR DESCRIPTION
**Pull request by frbuceta**
## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-connector-mssql) 👈

- [x] `npm test` passes on your machine
- New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](https://loopback.io/doc/en/contrib/style-guide-es6.html)
- [x] Commit messages are following our [guidelines](https://loopback.io/doc/en/contrib/git-commit-messages.html)

### CODE REVIEW

File: pretest.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ if (!process.env.MSSQL_DATABASE) process.env.MSSQL_DATABASE = global.getConfig()
 const catFileCmd = (isWin ? 'type ' : 'cat ') + sqlFileDir;
 
 const sqlcmd = catFileCmd + ' | ' + sqlDependencyDir + ' -s ' + process.env.MSSQL_HOST + ' -o ' + process.env.MSSQL_PORT +
-' -u ' + process.env.MSSQL_USER + ' -p ' + process.env.MSSQL_PASSWORD + ' -d ' + process.env.MSSQL_DATABASE;
+' -u ' + process.env.MSSQL_USER + ' -p "' + process.env.MSSQL_PASSWORD + '" -d ' + process.env.MSSQL_DATABASE;
```
Comment (raymondfeng) [inline pretest.js]: Does it make sense to quote all vars?

Comment (frbuceta) [reply pretest.js]: My database password is "yourStrong (!) Password" and without quotes, it doesn't work

Comment (raymondfeng) [reply pretest.js]: I understand. I meant to say if we should use `"` for user and database names too.

Comment (frbuceta) [reply pretest.js]: Sorry, I had not understood! Resolved

**Comment by frbuceta** [comment]
@raymondfeng what do you see?

**Pull request merged**